adapters/openweb/openwebtest/supplemental/wrong-impression-ext.json

```diff
@@ -0,0 +1,26 @@
+{
+  "mockBidRequest": {
+    "id": "unsupported-native-request",
+    "imp": [
+      {
+        "id": "unsupported-native-imp",
+        "video": {
+         "w": 100,
+          "h": 200
+        },
+        "ext": {
+          "bidder": {
+            "aid": "some string instead of int"
```

Comment:
You don't need a JSON test for invalid param values. That should  be covered in `params_test.go`